subsys/bluetooth/mesh/crypto_mtls.c

```diff
@@ -0,0 +1,639 @@
+/*
```

Comment:
files crypto_tc and crypto_mtls are mostly the same. I created the task to make security.c with common part and keep the only difference in crypto_tc and crypto_mtls